### PR TITLE
Add eunomia-bpf to eBPF Tools

### PR DESCRIPTION
**Pull request by yunwei37**
See https://github.com/eunomia-bpf/eunomia-bpf and https://ebpf.io/applications#eunomia-bpf.

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -223,6 +223,10 @@ If you are new to eBPF, you may want to try the links described as "introduction
 - [aya](https://github.com/aya-rs/aya) - A pure Rust library for writing, loading, and managing eBPF objects, with a focus on developer experience and operability. It supports writing eBPF programs in Rust and distributing library code over crates.io to share it between eBPF programs. Aya does not depend on libbpf.
 - [aya-template](https://github.com/aya-rs/aya-template) - Templates for writing BPF applications in Aya that can be used with [`cargo generate`](https://github.com/cargo-generate/cargo-generate).
 
+### eunomia-bpf
+
+- [eunomia-bpf](https://github.com/eunomia-bpf/eunomia-bpf) - A compile framework and runtime library for build, distribute, dynamically load and run CO-RE eBPF app in multi languages and Webassembly. It supports writing eBPF kernel code only to build simple CO-RE libbpf eBPF applications, writing kernel part in both BCC and libbpf style, and writing userspace in multiple languages in WASM module and distribute with simple JSON data or WASM OCI images. The runtime is based on libbpf only and is able to provide CO-RE to BCC style eBPF program without llvm dependence.
```
Comment (qmonnet) [inline readme.md:228]: Thanks a lot! It's a nice addition to the list.

I've got some comments on the grammar, please have a look below:

```suggestion
- [eunomia-bpf](https://github.com/eunomia-bpf/eunomia-bpf) - A compilation framework and runtime library to build, distribute, dynamically load, and run CO-RE eBPF applications in multiple languages and WebAssembly. It supports writing eBPF kernel code only (to build simple CO-RE libbpf eBPF applications), writing the kernel part in both BCC and libbpf styles, and writing userspace in multiple languages in a WASM module and distributing it with simple JSON data or WASM OCI images. The runtime is based on libbpf only and provides CO-RE to BCC-style eBPF programs without depending on the LLVM library.
```

The second sentence (`It supports...`) is long and difficult to read. I understand it comes from the items from the first list in https://github.com/eunomia-bpf/eunomia-bpf#readme, but could we make it easier to parse? If you want to keep all these items in the descriptions, maybe use inline numbers at least (`It supports 1) writing ..., 2) writing the kernel part..., and 3) writing userspace...`), with a separate sentence for packing/distributing?

Also `is able to provide CO-RE to BCC style eBPF program` -> Not sure what this means exactly, do you mean the programs are written in the style of BCC, but they use CO-RE and don't need LLVM? Is the BCC reference necessary here, or we should just mention CO-RE support?

Comment (yunwei37) [reply readme.md]: Thank you for the suggestions and for pointing out the issues with the grammar. I apologize for my poor English....

I have made the changes you suggested to make the description easier to read.

> Also is able to provide CO-RE to BCC style eBPF program -> Not sure what this means exactly, do you mean the programs are written in the style of BCC, but they use CO-RE and don't need LLVM? Is the BCC reference necessary here, or we should just mention CO-RE support?

Yes, I mean the programs are written in the style of BCC, but they use CO-RE and don't need LLVM.

Let me know if you have any further questions or comments! Thank you very much!

Comment (qmonnet) [reply readme.md]: > Thank you for the suggestions and for pointing out the issues with the grammar. I apologize for my poor English....

No need to apologise, I'm not a native speaker either. We're doing our best and doing grammar mistakes... That's also why we have reviews :)

Thanks for the update. Looks all good to me now!

**Review by qmonnet — APPROVED** [review]
Looks good to me now, thank you!

**Pull request merged**